containers/themes/CustomThemeModal.js

```diff
@@ -1,23 +1,25 @@
 import React, { useState } from 'react';
 import { c } from 'ttag';
 import PropTypes from 'prop-types';
-import { FormModal, Label, Alert, TextArea } from 'react-components';
+import { FormModal, PrimaryButton, Label, Alert, TextArea } from 'react-components';
 
-const CustomThemeModal = ({ onClose, onSave, theme: initialTheme = '', ...rest }) => {
+const CustomThemeModal = ({ onSave, theme: initialTheme = '', ...rest }) => {
     const [theme, setTheme] = useState(initialTheme);
 
     const handleChange = ({ target }) => setTheme(target.value);
-    const handleSubmit = () => onSave(theme);
+    const handleSubmit = async () => {
+        await onSave(theme);
```

Comment:
Can we add a success notification here?

Reply:
The notification for the custom theme is defined in `ThemesSection`. Like that we avoid calling `useNotifications` here